### PR TITLE
(ketarin) Migrated package

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description
Migrated package from @ferventcoder's repository.

## Motivation and Context
All packages from @ferventcoder's repository is meant to be moved here eventually.

reference issue #900 

## How Has this Been Tested?
Using the chocolatey test environment

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Migrated package (a package have been migrated from another repository)

## Checklist:

- [X] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this mean usually the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this mean usually the notes in the description of a package).
- [X] All files is up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [X] The added/modified package passed install/uninstall in the chocolatey test environment.
- [X] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/ferventcoder/chocolatey-packages)
- [Open Issues](https://github.com/ferventcoder/chocolatey-packages/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20ketarin) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [X] [more tags for ketarin](https://github.com/ferventcoder/chocolatey-packages/issues/106)
- [x] [PR Removal](https://github.com/ferventcoder/chocolatey-packages/pull/271)
- [X] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/ketarin/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+$toolsPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  packageName = $env:ChocolateyPackageName
+  file        = "$toolsPath\"
+  destination = $toolsPath
+}
+
+Get-ChocolateyUnzip @packageArgs
+Set-Content -Path "$toolsPath\Ketarin.exe.gui" -Value ""
```
Comment (majkinetor) [inline automatic/ketarin/tools/chocolateyInstall.ps1]: This isn't required any more AFAIK.

File: automatic/ketarin/Readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# [<img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/2963b7e22c35b0dfb637bb1d65ab74eb298bc66c/icons/ketarin.png" height="48" width="48" /> ketarin](https://chocolatey.org/packages/ketarin)
```
Comment (majkinetor) [inline automatic/ketarin/Readme.md]: Add Features from the site.

File: automatic/ketarin/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (AdmiringWorm) [reply automatic/ketarin/tools/chocolateyInstall.ps1]: No harm to have it though, so a gui shim is sure to be made both for new and older versions of chocolatey.

File: automatic/ketarin/Readme.md (hunk 2 above)
Comment (AdmiringWorm) [reply automatic/ketarin/Readme.md]: ah, I missed the entire feature section on the website. I'll get that updated.

Comment (AdmiringWorm) [reply automatic/ketarin/Readme.md]: updated

**Comment by gep13** [comment]
@AdmiringWorm I have merged your PR into @ferventcoder repository and added the `chocolatey` user to package, so this should be good to go.

**Comment by gep13** [comment]
@AdmiringWorm did this issue: https://github.com/ferventcoder/chocolatey-packages/issues/106 get taken care of during this migration?

**Comment by AdmiringWorm** [comment]
@gep13 I did add a couple of new tags because of that issue, but the package could always use more.

**Comment by gep13** [comment]
@AdmiringWorm ok, in which case, for now, I will go ahead and close that issue, as you have added some new ones, and it passed the review process.

**Pull request merged**